spongehome.go

```diff
@@ -26,23 +26,64 @@
 package main
 
 import (
-    "github.com/SpongePowered/SpongeHome/controllers"
-    "github.com/go-macaron/pongo2"
-    "gopkg.in/macaron.v1"
+	"fmt"
+	"github.com/SpongePowered/SpongeHome/controllers"
+	"github.com/go-macaron/pongo2"
+	"github.com/sethvargo/go-fastly"
+	"gopkg.in/macaron.v1"
+	"os"
+	"time"
 )
 
-func main() {
-    // Initialise Macaron
-    m := macaron.Classic()
-    m.Use(pongo2.Pongoer())
+func clearfastly() {
+	//clear fastly
+	if os.Getenv("FASTLY_KEY") != "" {
+		time.Sleep(10 * time.Second) //let the http server startup
+		fmt.Println("starting fastly client")
+		client, err := fastly.NewClient(os.Getenv("FASTLY_KEY"))
+		if err != nil {
+			fmt.Println(err)
+		}
+		services, err := client.ListServices(&fastly.ListServicesInput{})
+		FASTLY_NAME := ""
+		if os.Getenv("SPONGE_ENV") == "prod" {
+			FASTLY_NAME = "www.spongepowered.org"
+		} else if os.Getenv("SPONGE_ENV") == "staging" {
+			FASTLY_NAME = "www-staging.spongepowered.org"
+		}
+		for _, svc := range services {
+			if svc.Name == FASTLY_NAME {
```

Comment:
i had thought of that originally but from my looking around the API it looked like that could change if we reconfigured the settings for the url so i decided to just rely on the url itself for getting the active configuration